CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 ## Changelog
 
+* 7.2.0 Add support for `COLLATION` configuration parameter.
```

Comment:
Do we need to update the changelog manually (i.e. will semantic release update this for us automatically)?

Reply:
I can't tell. I'll ask Eli.

Reply:
Eli says this repo doesn't have semantic release yet.